src/rpc/methods/eth/filter/tipset.rs

```diff
@@ -11,14 +12,19 @@ use std::any::Any;
 #[allow(dead_code)]
 #[derive(Debug, PartialEq)]
 pub struct TipSetFilter {
-    id: FilterID,
-    max_results: usize,
+    pub id: FilterID,
+    pub max_results: usize,
+    pub collected: Option<ChainEpoch>,
```

Comment:
Same as `MempoolFilter`. Side note - can the logic between them can be somehow deduplicated?

Reply:
Each filter type needs to have this `collected` field to track at which point the results were collected